features/element-hiding.json

```diff
@@ -1065,6 +1065,20 @@
                     }
                 ]
             },
+            {
+                "domain": "cricbuzz.com",
+                "rules": [
+                    {
+                        "selector": "#loaderCon",
+                        "type": "modify-style",
+                        "values": [
+                            {
+                                "opacity": "100"
```

Comment:
@icodebyamanda  Is this better than using `hide` or `hide-empty`? If so, then `opacity` should be either a percentage or a value between 0 and 1, so `100%` or `1` is best here.

Reply:
thanks @tagawa - I changed the opacity - sorry for that mistake. For rather hide or hide-empty. I brought a related topic to our next monthly. We can then review the decision for that site.